docs/test_docs.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
```

Comment:
Is this still used?

Reply:
If not, it should probably be deleted and removed from the Docs job in BuildKite